packages/cli/src/ui/components/InputPrompt.tsx

```diff
@@ -42,15 +52,14 @@ export const InputPrompt: React.FC<InputPromptProps> = ({
   onClearScreen,
   config,
   slashCommands,
-  placeholder = '  Type your message or @path/to/file',
+  placeholder = '  Type your message or @path/to/file (Ctrl+V for images)',
```

Comment:
I'd suggest we don't include this in the placeholder unless @miguelsolorio has some good ideas of how to make this read well.